settings.php

```diff
@@ -41,24 +43,59 @@
     );
     $ADMIN->add('tools', $settings);
 
-    $settings->add(new admin_setting_configcheckbox(
-        'tool_excimer/excimerenable',
-        new lang_string('excimerenable', 'tool_excimer'),
-        '', 0));
+    if ($ADMIN->fulltree) {
+        $settings->add(
+            new admin_setting_configselect(
```

Comment:
I'm unable to test this, I don't see any of the plugin settings or report links anymore, not sure why

I think I'd prefer to not combine the settings and stick with a single enable but I'd like to test it first and see

Reply:
It has been converted back to a checkbox.